app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -48,8 +48,6 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     group_operation :suspend, 'Suspend Servers'
     group_operation :resume, 'Resume Servers'
 
-    specific_operation :remove_middleware_datasource, 'RemoveDatasource'
-
```

Comment:
there was a special handling for remove? but not for other operations?

Reply:
yes, it was introduced by me in #111, but it's wrong because the remove action can't be run on MW server because the controller is different (the operations works fine but the notifications are not)